spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -697,6 +697,7 @@ The method `BeanManager.getInterceptionFactory()` returns an `InterceptionFactor
 <T> InterceptionFactory<T> createInterceptionFactory(CreationalContext<T> ctx, Class<T> clazz);
 ----
 
+If type parameter `T` is not a class non-portable behavior results.
```

Comment:
What about...
> If the actual type parameter of the method is not a Java class, non-portable behavior results.

I think we should also add a similar note to API, `javax.enterprise.inject.spi.InterceptionFactory<T>`.